lib/AppInfo/Application.php

```diff
@@ -72,8 +73,8 @@ public function register(IRegistrationContext $context): void {
 	}
 
 	public function boot(IBootContext $context): void {
-		$context->injectFn(function (ITemplateManager $templateManager, IL10N $l10n, IConfig $config, CapabilitiesService $capabilitiesService) {
-			if (empty($capabilitiesService->getCapabilities())) {
+		$context->injectFn(function (ITemplateManager $templateManager, IL10N $l10n, IConfig $config, CapabilitiesService $capabilitiesService, PermissionManager $permissionManager) {
+			if (empty($capabilitiesService->getCapabilities()) || !$permissionManager->isEnabledForUser()) {
```

Comment:
Are capabilities not already empty when the app is disabled for the user? https://github.com/nextcloud/richdocuments/pull/2328

Reply:
This is where Collabora capabilities and Nextcloud capabilities might cause some confusion.  The CapabilitiesService returns the ones from Collabora and does not have such a check. 

The Nextcloud capabilities (which are usually accessed by an API call in scope of the current user) are limited however.